docs/website/src/eak-zoom-image/eak-zoom-image.ts

```diff
@@ -0,0 +1,127 @@
+import { ESLBaseElement } from '@exadel/esl/modules/esl-base-element/core';
+import { listen } from '@exadel/esl/modules/esl-utils/decorators/listen';
+import { attr } from '@exadel/esl/modules/esl-utils/decorators/attr';
+import { boolAttr } from '@exadel/esl/modules/esl-utils/decorators/bool-attr';
+import { memoize } from '@exadel/esl/modules/esl-utils/decorators/memoize';
+import { getTouchPoint } from '@exadel/esl/modules/esl-utils/dom/events/misc';
+
+import type { Point } from '@exadel/esl/modules/esl-utils/dom/events/misc';
+
+export class EAKZoomImage extends ESLBaseElement {
+  static is = 'eak-zoom-image';
+
+  @attr({ dataAttr: true })
```

Comment:
for better readability here and below, mb will we write it in one line?
```suggestion
  @attr({ dataAttr: true }) public src: string;
```